setup.py

```diff
@@ -24,6 +24,7 @@
     'install': [
         'cartopy',
         'cdo',
+        'cdsapi',
```

Comment:
I think you need this line also in `meta.yaml`.

Reply:
Ok, I added it.